.github/workflows/ci.yaml

```diff
@@ -0,0 +1,148 @@
+name: CI Tasks for Ory CLI
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: |
+          make test
+
+  docs:
+    name: Generate docs
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: |
+          mkdir -p docs/docs
+          make docs/cli
+          echo 'docsdir='"./docs" >> $GITHUB_ENV
+      - uses: actions/checkout@v2
+        with:
+          repository: ory/docs
+          path: ${{ env.docsdir }}
+          token: ${{ secrets.PAT }}
+      - env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          git config --global user.email "github-actions@github.com"
+          git config --global user.name "github-actions"
+          cd "$docsdir"
+          git add -A
+          git commit -a -m "autogen(docs): regenerate ory/cli reference" --allow-empty
+          git pull -ff
+          git push
+
+  release:
+    name: Generate release
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - test
+    steps:
+      - uses: ory/ci/releaser@master
+        with:
+          token: ${{ secrets.PAT }}
```

Comment:
Can we scope the permissions needed for this token?

Reply:
Same as above.